docs/fundamentals/client/client.md

```diff
@@ -760,6 +760,34 @@ None
 
 ````
 
+A Client connects to a Flow that processes Documents in an asynchronous and very distributed way. This means that the order of the Flow processing the requests may not be the same order as the Client sending the requests.
```

Comment:
Is there a way to pass this parameter from a 3rd party client?

Reply:
No, this is gonna be ready in the `streamer` for the CustomGateway. We can add it later I would say

Reply:
Mostly external clients do not have this `streaming` capability so easily implemented, so I think is not so so important